_posts/en/posts/2021-09-13-release-22.0.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Bitcoin Core 22.0 released
+name: blog-release-22.0
+id: en-blog-release-0.22.0
+lang: en
+type: posts
+layout: post
+
+## If this is a new post, reset this counter to 1.
+version: 1
+
+## Only true if release announcement or security annoucement.  English posts only
+announcement: 1
+
+excerpt: >
+  Bitcoin Core 22.0 is now available.
+---
+Bitcoin Core version 22.0 is now available for [download][download
+page].  See the [release notes][] for more information about the many
+new features and bug fixes in this release.
+
+If have any questions, please stop by the #bitcoin IRC chatroom
+([IRC][irc], [web][web irc]) and we’ll do our best to help you.
+
+[release notes]: /en/releases/22.0/
+[IRC]: irc://irc.freenode.net/bitcoin
```

Comment:
that should probably be updated to libera (and the next line, too)

Reply:
Updated